src/excmds.ts

```diff
@@ -2507,9 +2507,8 @@ export async function undo(item = "recent"): Promise<number> {
             : !isNaN(parseInt(item, 10))
             ? s => (s.tab || s.window).sessionId === item
             : s => {
-                  throw new Error("blah")
+                  throw new Error(`[undo] Invalid argument: ${item}. Must be one of "recent, "tab", "tab_strict" or "window"`)
```

Comment:
this isn't quite true as you can do `:undo [sessionId]` (a number, accessible to humans via the undo completions). Too tired to word it better now : )

Reply:
Clarified the error message.